db/migrate/20160413121293_create_exemptions.rb

```diff
@@ -21,5 +21,10 @@ def change
 
     add_foreign_key :flood_risk_engine_enrollments_exemptions,
                     :flood_risk_engine_exemptions, column: :exemption_id
+
+    add_index :flood_risk_engine_enrollments_exemptions,
+              [:enrollment_id, :exemption_id],
+              unique: true,
+              name: "fre_enrollments_exemptions_enrollment_id_exemption_id"
   end
```

Comment:
Once a migration has been committed to the repository it shouldn't be modified, unless something is broken.